GliaWidgets/Public/Glia/Glia.Deprecated.swift

```diff
@@ -1,8 +1,29 @@
 import GliaCoreSDK
 
 extension Glia {
+    /// Deprecated, use the `configure` method that provides a `Result` in its completion instead.
```

Comment:
if you don't mind, could we add this deprecation method to the end of list? :) so all methods will be sorted from older to newest :)

Reply:
Moved c4a3367307e32d67841b1175a1631f284dac7c85